### DISCUSSION TITLE
Notification bug?

### DISCUSSION THREAD

**Discussion by OscarSheng** [community/community#21594] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/c/7cfb6d511cd05a8d43ed6d524bf782d1f6dc6eb1.png" data-download-href="/uploads/short-url/hPDSWHELcNE8C3Toh3ytUxfhfb3.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/36452729/181065760-c182f5c0-f29e-4aa0-a995-5d10ea4d4ba8.png" alt="image" data-base62-sha1="hPDSWHELcNE8C3Toh3ytUxfhfb3" width="690" height="384" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7cfb6d511cd05a8d43ed6d524bf782d1f6dc6eb1_2_690x384.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7cfb6d511cd05a8d43ed6d524bf782d1f6dc6eb1_2_1035x576.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7cfb6d511cd05a8d43ed6d524bf782d1f6dc6eb1_2_1380x768.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7cfb6d511cd05a8d43ed6d524bf782d1f6dc6eb1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1772×988 33.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
There is no notification in the inbox, but the blue dot still there, is this a bug?

**Comment by Flouse** [comment]
<p>I think it is a bug.<br>
My account faces the same bug.</p>
<hr>
<p>see also:</p><aside class="quote" data-post="6" data-topic="142727">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/no-new-notification-but-inbox-show-1/142727/6">No new notification but inbox show 1</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    I don’t have an estimate for that I’m afraid, but I can tell you they’re already working on it. That said, it is not a straightforward issue so honestly it may take some time.
  </blockquote>
</aside>

**Comment by Flouse** [comment, marked as answer]
<p>From GitHub Support:</p>
<blockquote>
<p>We have seen things like this happen before. Can you try a couple of things that should work.</p>
<p>Select the checkbox and see if there is an option to select all and mark all as done/read. If that is not an option, I have had success before with the REST API.</p>
<p>You can use the details from that API endpoint to find the unread notification and then the one below to mark them as read.</p>
<p><a href="https://docs.github.com/en/rest/reference/activity#mark-notifications-as-read" class="inline-onebox" rel="noopener nofollow ugc">Activity - GitHub Docs</a></p>
<p>The <code>curl</code> command below will mark all of your notifications as read and with any luck will take care of that phantom message for you.</p>
<pre><code class="lang-auto">curl -X PUT -H "Accept: application/vnd.github.v3+json" -H "Authorization: bearer {token}" https://api.github.com/notifications -d '{"last_read_at":"2021-05-28T06:53:00"}'
</code></pre>
<p>You will need a Personal Access Token (PAT) to access the API from the command line. The PAT is used in place of the password to authenticate. To create a PAT follow the steps in the link below.</p>
<p><a href="https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token" rel="noopener nofollow ugc">https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token</a></p>
<p>Below that are the instructions to access GitHub from the command line using the PAT.</p>
<p>I hope this helps and please let me know what else I can assist with!</p>
<p>Cheers,<br>
J.J.</p>
</blockquote>
